net/dhtd/files/dhtd.config

```diff
@@ -15,6 +15,9 @@ config dhtd
 	list peer 'bttracker.debian.org:6881'
 	list peer 'router.bittorrent.com:6881'
 
+## Execute a script for each result
+#	option execute '/root/on_result.sh'
```

Comment:
This is related to its update or not? Would be good to mention that in the commit description.

Reply:
Done